### ISSUE TITLE
New rule: operation descriptions should be in third person

### ISSUE THREAD

**Issue by veronicagg** [Azure/azure-openapi-validator#58]
Please find details at https://github.com/Azure/autorest/issues/1994

**Comment by veronicagg** [comment]
@sptramer is this something you'd like to address on your side?

**Comment by sptramer** [comment]
Yes, please create this as an issue on azure/azure-openapi-refchecker. This will be a lower-priority item for us as we want to see if more basic ref rules will improve quality before getting into NLP such as tense and perspective analysis.
